test/unit/extract/xbrl_test.py

```diff
@@ -130,13 +125,16 @@ def test_xbrl2sqlite(settings, forms, mocker, tmp_path):
             form,
             mock_datastore,
             output_path=PudlPaths().output_dir,
-            sql_path=tmp_path / f"ferc{form.value}_xbrl.sqlite",
+            sql_path=PudlPaths().output_dir / f"ferc{form.value}_xbrl.sqlite",
             batch_size=20,
             workers=10,
         )
 
 
-def test_xbrl2sqlite_db_exists_no_clobber(mocker):
+def test_xbrl2sqlite_db_exists_no_clobber(mocker, live_dbs):
```

Comment:
I know I wrote these tests to actually hit disk, but that was probably a mistake. A unit test really shouldn't have to know if `--live-dbs` is set, because, well. It's a unit test.

I think that means we either need to: 
- mock out `PudlPaths().various_path_attributes` to return mocked Path objects, which we can then mock th `touch()`, `exists()` etc. API on
- mock out `PudlPaths` to always use tmp dir for output in unit tests, no matter what `--live-dbs` says - probably in `unit/conftest.py`

Do you have opinions or preference here? Personally I think the second one is harder to mess up in the future but I can definitely see an argument for "unit tests shouldn't be touching disk at all."

Reply:
PudlPaths is instantiated from env variables so we could just temporarily set those and run whatever isolated tests we need to.